src/execution_plan/ops/op_conditional_traverse.c

```
@@ -1,440 +1,78 @@
-
-	if (op->optional) {
-		CondTraverse_MakeOptional ((OpCondTraverse*)clone) ;
-	}
-
-	return clone ;
+static void CondTraverseFree(OpBase *opBase) {
+	OpCondTraverse *op = (OpCondTraverse *)opBase;
+	if(op->output_records) array_free(op->output_records);
+	if(op->r) OpBase_DeleteRecord(op->r);
+	if(op->relationFilter) FilterNode_Free(op->relationFilter);
 }
```

Cppcheck (2.21.0)

[error] Memory leak

(memleak)